apps/condo/domains/billing/utils/period.js

```diff
@@ -0,0 +1,31 @@
+function getPeriodMessage (period, locale = 'default') {
+    if (!period || !locale) return
+    const date = new Date(period)
+    if (isNaN(date.getMonth())) return
+    const fullMonth = date.toLocaleString(locale, { month: 'long' })
+    return `${fullMonth} ${date.getFullYear()}`
+}
+
+function getPreviousPeriods (period, totalAmount = 3) {
```

Comment:
can u use moment.js or date-fns for this periods generation?

Reply:
replaced with luxon 👍

Reply:
Why?